prebid-mobile/code-integration-ios.md

```diff
@@ -62,55 +78,92 @@ Embed the ad unit registration in a try-catch block to catch all the exceptions
 
 [PBLogManager setPBLogLevel:PBLogLevelAll];
 
-// 1. Create the ad units with ad unit ids and add sizes for banner ad units
-PBBannerAdUnit *__nullable adUnit1 = [[PBBannerAdUnit alloc] initWithAdUnitIdentifier:@"YOUR-AD-UNIT-ID-HERE" andConfigId:@"YOUR-CONFIG-ID-HERE"];
+/**
+ * 1. Create the ad units and add sizes for banner ad units.
+ *
+ * Replace @"PREBID-MOBILE-SLOT-ID" with the unique ad slot identifier
+ * you defined when you registered the ad unit with Prebid Mobile. 
+ *
+ * Replace @"PREBID-SERVER-CONFIGURATION-ID" with the ID of 
+ * your Prebid Server demand partner configuration.
+ */ 
+PBBannerAdUnit *__nullable adUnit1 = [[PBBannerAdUnit alloc] initWithAdUnitIdentifier:@"PREBID-MOBILE-SLOT-ID" andConfigId:@"PREBID-SERVER-CONFIGURATION-ID"];
 [adUnit1 addSize:CGSizeMake(300, 250)];
 
-// 2. Set targeting parameters for the ad units (Optional)
+/**
+ * 2. Set targeting parameters for the ad units (Optional).
+ */ 
 [[PBTargetingParams sharedInstance] setAge:25];
 [[PBTargetingParams sharedInstance] setGender:PBTargetingParamsGenderFemale];
 
-// 3. Register the ad units with Prebid Mobile to start bid fetching process
+/**
+ * 3. Register the ad units with Prebid Mobile to start bid fetching process.
+ *
+ * Replace @"PREBID-SERVER-ACCOUNT-ID" with your Prebid Server account ID.
+ */ 
 [PrebidMobile registerAdUnits:@[adUnit1, adUnit2]
-          		withAccountId:kAccountId
+          		withAccountId:@"PREBID-SERVER-ACCOUNT-ID"
                		 withHost:PBServerHostAppNexus
```

Comment:
Shouldnt we detail about the Host too??? currently the document defaults to AppNexus